### PR TITLE
improving contrib guide and readme

### PR DESCRIPTION
**Pull request by iagocarmona**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

Improving contrig guide and readme adding PR steps 

### Related issue

- #407

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks @iagocarmona! We'll check this out in the coming days :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -203,11 +203,11 @@ When making a contribution, adhering to the [GitHub flow](https://guides.github.
...
    # Clone your fork of the repo into the current directory
-   git clone https://github.com/<your-username>/<repo-name>
+   git clone https://github.com/<your-username>/activist.git
    # Navigate to the newly cloned directory
-   cd <repo-name>
+   cd activist
    # Assign the original repo to a remote called "upstream"
-   git remote add upstream https://github.com/<upstream-owner>/<repo-name>
+   git remote add upstream https://github.com/activist-org/activist.git
    ```
```
Comment (to-sta) [inline CONTRIBUTING.md:211]: I would add a line after the upstream:

Now, if you run `git remote -v ` you should see two remote repositories named:
* origin (forked repository)
* upstream (activist repository)

File: README.md [hunk 2]
```diff
@@ -125,16 +125,20 @@ Note that the Figma file above is the [public facing designs](https://www.figma.
...
+3. [Fork](http://help.github.com/fork-a-repo/) the repo, clone your fork, and configure the remotes:
 
 ```bash
-git clone https://github.com/activist-org/activist.git
+# Clone your fork of the repo into the current directory
+git clone https://github.com/<your-username>/activist.git
+# Navigate to the newly cloned directory
+cd activist
+# Assign the original repo to a remote called "upstream"
+git remote add upstream https://github.com/activist-org/activist.git
```
Comment (to-sta) [inline README.md:136]: > I would add a line after the upstream:
> 
> Now, if you run `git remote -v ` you should see two remote repositories named:
> 
> * origin (forked repository)
> * upstream (activist repository)

Same here :)

Comment (iagocarmona) [reply README.md:136]: @to-sta Resolved

**Review by to-sta — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (andrewtavis) [reply CONTRIBUTING.md:211]: Good thought, @to-sta!

**Review by andrewtavis — APPROVED** [review]
Beautiful stuff here, @iagocarmona! Thanks so much for the care you've put into this! I think this is gonna be great 😊

**Pull request merged**